.github/workflows/main.yml

```diff
@@ -49,14 +49,13 @@ jobs:
         uses: actions/checkout@v4
         with:
           repository: gavalian/hipo
-          ref: 4.0.1 # if you want to use a fixed tag; remove if you want to use `main`
+          ref: 20094876df040ec234dd376fa527cf58442b775a
```

Comment:
@dglazier would you prefer to use a fixed version of HIPO for CI tests, or use its `master` branch? The `master` branch means you'll always be testing the latest version, whereas using a fixed version means someone has to remember to periodically update the `ref` here.

Reply:
I think with the fixed version. Actually I am not sure why, when we went to an external hipo I did not just make it a submodule. This way the version is fixed in any clas12root tag and should always work. Perhaps it makes sense to do it that way ?

Reply:
That's fine too. Would you like me to use a submodule in this PR? This would then be more consistent with your other dependencies.

Reply:
Yes I think that would be best, thanks.